feat(content): New Coalition License description (#10584)
endless-sky/endless-sky: data/coalition/coalition outfits.txt

outfit "Coalition License"
	category "Licenses"
	thumbnail "outfit/coalition license"
	description "A digital chip in the form of a stylized badge, this document signifies the permission of the owner to browse, employ and deal in transactions involving civilian Coalition technology."

> 	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the requirements that need to be met for someone to be issued one, starting with the third Civil Goods and Fares Act, just a few decades after the War of Independence, all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which two hundred years ago made it much more flexible to acquire such a license."

Reviewer: "...made it much more flexible..." reads as being somewhat unclear to me (what is being flexed?) This suggestion also changes the first mention of "requirements" so it doesn't appear twice in the same sentence with the suggested change.
```suggestion
	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to be issued one, starting with the third Civil Goods and Fares Act, just a few decades after the War of Independence, all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which two hundred years ago made the acquisition requirements for such a license much more flexible."
```

> 	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the requirements that need to be met for someone to be issued one, starting with the third Civil Goods and Fares Act, just a few decades after the War of Independence, all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which two hundred years ago made it much more flexible to acquire such a license."

Reviewer 2: This last sentence runs on a lot.

Author: Any idea on how to break it up?

> 	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to be issued one, starting with the third Civil Goods and Fares Act, just a few decades after the War of Independence, all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which two hundred years ago made the acquisition requirements for such a license much more flexible."

Reviewer 2: For some reason GitHub didn't let me suggest changes above.
```suggestion
	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to obtain a license. This argument started with the third Civil Goods and Fares Act, created just a few decades after the War of Independence, and has continued all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which was passed two hundred years ago and made the acquisition requirements for such a license much more flexible."
```
The second part probably still needs a little breakup, but I have no idea where to put it.

> 	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to obtain a license. This argument started with the third Civil Goods and Fares Act, created just a few decades after the War of Independence, and has continued all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, which was passed two hundred years ago and made the acquisition requirements for such a license much more flexible."

Reviewer 3: An attempt at breaking up that last sentence:

```suggestion
	description "	There is a history of controversy surrounding this license, as throughout the millennia several Heliarch laws have been in a back and forth regarding the conditions that need to be met for someone to obtain a license. This argument started with the third Civil Goods and Fares Act, created just a few decades after the War of Independence. It has continued all the way to the relatively recent twenty-sixth addendum to the second article of the Ninth Rectification on Commerce and Traffic, passed two hundred years ago, which made the acquisition requirements much more flexible."
```
